Packs/BarracudaEmailProtection/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### Barracuda Email Protection Parsing Rule
+
+- Fixed an issue where the **Barracuda Email Protection** parsing rule was not located in the correct path.
```

Comment:
this is not relevant for customers
instead, let's say something about the change in the parsing rule itself